metrics/timers_test.go

```diff
@@ -0,0 +1,75 @@
+package metrics
+
+import (
+	"context"
+	"testing"
+
+	"gx/ipfs/QmNVpHFt7QmabuVQyguf8AbkLDZoFh7ifBYztqijYT1Sd2/go.opencensus.io/stats/view"
+	"gx/ipfs/QmPVkJMTeRC6iBByPWdrRkD3BE5UXsj5HPzb4kPqL186mS/testify/assert"
+	"gx/ipfs/QmPVkJMTeRC6iBByPWdrRkD3BE5UXsj5HPzb4kPqL186mS/testify/require"
+)
+
+func TestTimerSimple(t *testing.T) {
+	assert := assert.New(t)
+
+	ctx := context.Background()
+	reg := NewRegistry()
+	assert.Equal(len(reg.measures), 0)
+
+	testTimer := reg.NewTimer("testName", "testDesc")
+	// some view state is kept around after tests exit, doing this to clean that up.
+	// e.g. view will remain registered after a test exits.
+	defer view.Unregister(testTimer.view)
```

Comment:
Won't the registry be garbage collected and take everything with it? Why is there no unregistration in the other tests?

Reply:
I did not observe the garbage collector running between the tests in these files, without this line the code on line 41 panics.